source/helpers/is-low-quality-comment.test.ts

```diff
@@ -1,28 +1,43 @@
-import {test, assert} from 'vitest';
+import {test, assert, describe} from 'vitest';
 
 import isLowQualityComment from './is-low-quality-comment';
 
-test('isLowQualityComment', () => {
-	assert.isTrue(isLowQualityComment('+1'));
-	assert.isTrue(isLowQualityComment('+1!'));
-	assert.isTrue(isLowQualityComment('+10'));
-	assert.isTrue(isLowQualityComment('+9000'));
-	assert.isTrue(isLowQualityComment('-1'));
-	assert.isTrue(isLowQualityComment('👍'));
-	assert.isTrue(isLowQualityComment('👍🏾'));
-	assert.isTrue(isLowQualityComment('me too'));
-	assert.isTrue(isLowQualityComment('ditto'));
-	assert.isTrue(isLowQualityComment('Dito'));
-	assert.isTrue(isLowQualityComment('following'));
-	assert.isTrue(isLowQualityComment('please update!'));
-	assert.isTrue(isLowQualityComment('please update 🙏🏻'));
-	assert.isTrue(isLowQualityComment('same issue'));
-	assert.isTrue(isLowQualityComment('this same issues'));
-	assert.isTrue(isLowQualityComment('same question'));
-	assert.isTrue(isLowQualityComment('any updates there?'));
+describe('isLowQualityComment', () => {
+	test.each([
+		'+1',
+		'+1!',
+		'+10',
+		'+9000',
+		'-1',
+		'👍',
+		'👍🏾',
+		'me too',
+		'ditto',
+		'Dito',
+		'following',
+		'please update!',
+		'please update 🙏🏻',
+		'same issue',
+		'this same issues',
+		'same question',
+		'any updates there?',
+		'any news?',
+		'+++ !!!\nThanks !!!',
+		'+1\nThx for your job!',
+		'+1 and thank you for your work',
+		'Same here, please update, thanks',
+		'Same here! Please update, thank you.',
+	])('%s', text => {
+		assert.isTrue(isLowQualityComment(text));
+	});
 
-	assert.isFalse(isLowQualityComment('+1\n<some useful information>'));
-	assert.isFalse(isLowQualityComment('Same here. <some useful information>'));
-	assert.isFalse(isLowQualityComment('Same here, please update, thanks'));
-	assert.isFalse(isLowQualityComment('Same here! Please update, thank you.'));
+	test.each([
+		'+1\n<some useful information>',
+		'Same here. <some useful information>',
+		'Same here on v1.2',
+		'Thanks!',
+		'Thank you 👍',
```

Comment:
Why would these not be considered low quality?

Reply:
Isn't the intent of this PR to hide "Thanks"?

Reply:
Only saying "thanks" while pushing a bug report or feature request sounds like "thanks if this gets implemented", which is rather rude / demanding and thus low quality.

Saying "thanks" on its own, e.g. when a bug is fixed, is much more rewarding and positive.

Reply:
> when a bug is fixed

I think that's what matters though, not the exact wording. I'm not sure it's worth implementing a check to exclude those though

Reply:
@FloEdelmann should we block "thanks" like we do other words or can we close this PR?

Reply:
I believe we should not block "thanks". So maybe closing is better in this case.